homeassistant/components/evil_genius_labs/__init__.py

```diff
@@ -0,0 +1,99 @@
+"""The Evil Genius Labs integration."""
```

Comment:
Please exclude the init module from coverage calculation or add tests.

Reply:
The light test relies on all functionality in `__init__.py` and so this is almost fully tested.

Reply:
There's no test for unloading the entry.